Api/Filter/FilterInterface.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the DunglasJsonLdApiBundle package.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Dunglas\JsonLdApiBundle\Api\Filter;
+
+/**
+ * Filters applicable on a resource.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+interface FilterInterface
+{
+    const STRATEGY_EXACT = 'exact';
```

Comment:
I don't think that's the place of this constant since it's not something "abstract" but related to an implementation. What about an `applyFilter` method that would extract the filtering logic in that class ?